src/compiler/resolver_method.cc

```diff
@@ -2306,7 +2307,13 @@ ir::Node* MethodResolver::_resolve_call_target(ast::Node* target_node,
       }
       if (method_node->is_static()) {
         check_sdk_protection(name, target_node->range(), method_node->range());
-        return _new ir::ReferenceMethod(method_node, range);
+        // We special case the identical top-level method, because we want to
+        // turn calls to that into a special bytecode.
+        if (name == Symbols::identical) {
```

Comment:
We only want to replace `identical` calls to the core method.
(This is different to `assert` which is a keyword like function).
The holder check is probably not necessary, but this way we ensure that we don't refer to a static `Foo.identical` inside a class.
```suggestion
        if (name == Symbols::identical && method_node->is_runtime_method() && method_node->holder == null) {
```

Reply:
We could also check for the shape here. However, there really should only be one `identical` in the core libraries, so we should be safe.

Reply:
Done.